.github/workflows/QPE.yml

```diff
@@ -0,0 +1,33 @@
+name: QPE_example
```

Comment:
My thought is we likely shouldn't have GitHub actions dedicated to individual example pipelines, just the pipeline definition and README are fine here